joomla/joomla-cms
# Update default.php to fix missing space between img attributes

Pull Request for Issue #14893 .

### Summary of Changes
Added the correct space between the attributes inside the banner elements


### Testing Instructions
Overwrite the default.php in a joomla 3.6.5 with the fixed one. Try the steps from issue #14893 and check the HTML source. The spaces should be now correct. As an additional test you can run an HTML syntax check against the section and it now validated correctly.


### Expected result
img src="banners/banner_example_468x60.gif" alt="486x60" width="486" height="60"/


### Actual result
img src="banners/banner_example_468x60.gif" alt="486x60"width ="486"height ="60"/


### Documentation Changes Required
See source code
